containers/run_hydra.sh

```diff
@@ -22,7 +22,13 @@ else
   STORE="./store"
 fi
 
-SRV="$2"
+if [ "$2" = "" ] ; then
```

Comment:
Our bash script guidelines suggest using a 'case' statement instead of 'if' - 'elif' structure.
If you insist on using 'if', guidelines also suggest using [ -z "$VAR" ] for empty testing.

https://ssrc.atlassian.net/wiki/spaces/SP/pages/242483267/Bash+script+style

Reply:
Done